internal/bminventory/inventory.go

```diff
@@ -6158,6 +6174,178 @@ func (b *bareMetalInventory) V2UpdateHostIgnitionInternal(ctx context.Context, p
 	return &h.Host, nil
 }
 
+// getOVERegistriesConf returns the registry configuration for OVE (OpenShift Virtualization Environment) images.
+// This configuration maps all required registries to a local registry that will be available in the disconnected environment.
+func getOVERegistriesConf() string {
+	return `[[registry]]
+location = "quay.io/openshift-release-dev/ocp-release"
+
+[[registry.mirror]]
+location = "registry.appliance.com:5000/openshift/release-images"
+
+[[registry]]
+location = "quay.io/openshift-release-dev/ocp-v4.0-art-dev"
+
+[[registry.mirror]]
+location = "registry.appliance.com:5000/openshift/release"
+
+[[registry]]
+location = "registry.redhat.io"
+
+[[registry.mirror]]
+location = "registry.appliance.com:5000"
+
+[[registry]]
+location = "registry.openshift.com"
+
+[[registry.mirror]]
+location = "registry.appliance.com:5000"
+
+[[registry]]
+location = "quay.io"
+
+[[registry.mirror]]
+location = "registry.appliance.com:5000"
+`
+}
+
+// GenerateOVEIgnition generates the unconfigured-ignition content for OVE images
+// using openshift-install agent create unconfigured-ignition --interactive command
+func (b *bareMetalInventory) GenerateOVEIgnition(ctx context.Context, infraEnv *common.InfraEnv) (string, error) {
+	log := logutil.FromContext(ctx, b.log)
+	log.Infof("GenerateOVEIgnition called for infraEnv %s", *infraEnv.ID)
+
+	osImage, err := b.osImages.GetOsImageOrLatest(infraEnv.OpenshiftVersion, infraEnv.CPUArchitecture)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to get OS image for OVE ignition generation")
+	}
+
+	tempDir, err := os.MkdirTemp("", "ove-ignition-*")
```

Comment:
Don't use temp dir. Use our data dir instead. I believe tmp isn't writable in all environments.